mysite/general/secretPath.py

```diff
@@ -9,35 +9,72 @@
 
 skillsToReview_RightHand = ["Mining", "Choppin", "Fishing", "Catching", "Trapping"]  #, "Worship"]
 
-def getHandsAdviceGroup(maestros, beginners):
+def getHandsAdviceGroupCatchUp(maestros, beginners):
+    # This function returns the skills maestros aren't best in.
     janky_skills = maestros_goal_levels(maestros)
-    tier = f"{len(janky_skills) or 6}/{len(skillsToReview_RightHand)}"
+    zero_count = sum(1 for skill, (_, _, next_highest_non_maestro_char) in janky_skills.items() if not next_highest_non_maestro_char)
+    if zero_count == 5:
+        # No maestros ahead in any skill. I think?
+        return
+    tier = f"{zero_count}/{len(skillsToReview_RightHand)}"
     hands_pre_string = create_hands_pre_string(janky_skills, maestros, beginners, tier)
 
     post_string = ""
     if "Worship" in janky_skills:
         post_string = "Worship matters moreso for Species Epoch than Right Hand. Don't steal charge away from this character!"
 
-    advices = [
-        Advice(
-            label=maestro.character_name,
-            picture_class=skill,
-            progression=maestro.skills[skill],
-            goal=goal,
-        )
-        for skill, (maestro, goal) in janky_skills.items()
-    ]
+    advices = []
+
+    for skill, (maestro, goal, next_highest_non_maestro_char) in janky_skills.items():
+        if not next_highest_non_maestro_char:
+            advices.append(Advice(
+                label=maestro.character_name,
+                picture_class=skill,
+                progression=maestro.skills[skill],
+                goal=goal,
+            ))
+
     if advices:
         session_data.account.alerts_AdviceDict['General'].append(Advice(
             label=f"Your {{{{ Maestro|#secret-class-path }}}} isn't best in {len(advices)} Skill{pl(advices)}",
             picture_class='right-hand-of-action',
         ))
 
     group = AdviceGroup(
-            tier="", pre_string=hands_pre_string, post_string=post_string, advices=advices
-        )
+        tier="", pre_string=hands_pre_string, post_string=post_string, advices=advices
+    )
+    return group
+
+
+def getHandsAdviceGroupStayAhead(maestros):
+    # This function returns the skills maestros are best in.
+    janky_skills = maestros_goal_levels(maestros)
+    zero_count = sum(1 for skill, (_, _, next_highest_non_maestro_char) in janky_skills.items() if not next_highest_non_maestro_char)
+    if zero_count == 5:
+        # No maestros ahead in any skill. I think?
+        return
+    tier = f"{zero_count}/{len(skillsToReview_RightHand)}"
+    hands_pre_string = f"Your Maestro is the highest level in {5-zero_count} Right Hand skills, but others can overtake"
```

Comment:
Same note here. 
For consistency with the other Right Hands group, I'd like to see `already best in X/Y Right Hand Skills`